alluxio/posix/delegate.py

```diff
@@ -0,0 +1,14 @@
+import os
+from alluxio.posix import fileimpl
+config_manager = fileimpl.ConfigManager("../../config/ufs_config.yaml")
```

Comment:
The config path cannot be hard-coded in the code. It must have the feasibility of dynamic configuration, such as environment variables and API injection.